db_Test.R

```diff
@@ -0,0 +1,18 @@
+library(exasol)
+
+drv <- "/home/thomas/Downloads/EXASOL_ODBC-6.2.7/lib/linux/x86_64/libexaodbc-uo2214lv2.so"
```

Comment:
Is this file intended to be checked in? If so, maybe we can parameterize this path.

Reply:
Ops, good catch. I need to remove it.